tests/test_notify_slack.py

```diff
@@ -42,18 +47,41 @@
 from .general_generators import any_https_url
 from .stac_generators import any_dataset_id, any_dataset_prefix, any_dataset_version_id
 
+STEP_FUNCTION_START_DATE = round(
+    datetime(
+        2001, 2, 3, hour=4, minute=5, second=6, microsecond=789876, tzinfo=timezone.utc
+    ).timestamp()
+    * 1000
+)
+STEP_FUNCTION_STOPDATE = round(
```

Comment:
You could just do `STEP_FUNCTION_STOPDATE = STEP_FUNCTION_START_DATE + some_number` here.